cards enhancements

MasonZhang-MZ opened the pull request:
1. [align the cards with the figures #49](https://github.com/UBC-MDS/DSCI-532_2025_12_bank-marketing/issues/49)
2. [proportion subscribed to percentage #50](https://github.com/UBC-MDS/DSCI-532_2025_12_bank-marketing/issues/50)
3. [average contacts: new line #52](https://github.com/UBC-MDS/DSCI-532_2025_12_bank-marketing/issues/52)
4. [colors in the upper cards #53](https://github.com/UBC-MDS/DSCI-532_2025_12_bank-marketing/issues/53)
5. [color subscribed to match legend #54](https://github.com/UBC-MDS/DSCI-532_2025_12_bank-marketing/issues/54)

![image](https://github.com/user-attachments/assets/907c7933-f30c-442b-9b1a-1d96e996bae1)

MasonZhang-MZ committed:
cards enhancements

JiaQuanLim commented on src/components.py:
```diff
@@ -32,20 +77,39 @@ def create_plot_card(plot_id):
+        dbc.Col(create_stat_card([
+            "Average Last Contact",
+            html.Br(),
+            "Duration",
```
```suggestion
            "Duration (Seconds)"
```
I suggest we move the (Seconds) to the same line as Duration:
![image](https://github.com/user-attachments/assets/3402fcbc-c27f-4525-9d2a-3987b63abced)

JiaQuanLim reviewed (changes requested):
Hello Mason, just one small admendment, do you think it would be better if we move the (Seconds) to the same line as Duration? I have added my suggestion to the code.

MasonZhang-MZ committed:
Update src/components.py

that's a good idea, makes the overall display more consistent, thanks for the suggestion!

Co-authored-by: Joseph Lim <104155632+JiaQuanLim@users.noreply.github.com>

MasonZhang-MZ commented:
> Hello Mason, just one small admendment, do you think it would be better if we move the (Seconds) to the same line as Duration? I have added my suggestion to the code.

Had the same debate when I was editing, but I think you are right, same line would look a bit better. Commit approved.

JiaQuanLim commented on src/components.py:
```diff
@@ -32,20 +77,39 @@ def create_plot_card(plot_id):
+            "Average Last Contact",
+            html.Br(),
+            "Duration (Seconds)"
+            html.Br(),
```
```suggestion
```

JiaQuanLim commented on src/components.py:
```diff
@@ -32,20 +77,39 @@ def create_plot_card(plot_id):
+            html.Br(),
+            "Duration (Seconds)"
+            html.Br(),
+            "(Seconds)"
```
```suggestion
```

MasonZhang-MZ committed:
Update src/components.py

Co-authored-by: Joseph Lim <104155632+JiaQuanLim@users.noreply.github.com>

MasonZhang-MZ committed:
Update src/components.py

Co-authored-by: Joseph Lim <104155632+JiaQuanLim@users.noreply.github.com>
